packages/flutter/lib/src/material/material_state.dart

```diff
@@ -257,22 +266,47 @@ abstract class MaterialStateMouseCursor extends MouseCursor implements MaterialS
   /// disabled, the cursor resolves to [SystemMouseCursors.basic].
   ///
   /// This cursor is the default for many Material widgets.
-  static const MaterialStateMouseCursor clickable = _ClickableMouseCursor();
+  static const MaterialStateMouseCursor clickable = _SpecifyDisabledMouseCursor(
+    usualCursor: SystemMouseCursors.click,
+    disabledCursor: SystemMouseCursors.basic,
+    name: 'clickable',
+  );
+
+  /// A mouse cursor for material widgets related to text, which resolves differently
+  /// when the widget is disabled.
+  ///
+  /// By default this cursor resolves to [SystemMouseCursors.text]. If the widget is
+  /// disabled, the cursor resolves to [SystemMouseCursors.basic].
+  ///
+  /// This cursor is the default for many Material widgets.
+  static const MaterialStateMouseCursor textable = _SpecifyDisabledMouseCursor(
```

Comment:
I guess textable is like clickable (but I don't think textable is a word).  Would it be OK to just call this "text"?

Reply:
I agree with the concern, but "text" can confuse people with `SystemMouseCursors.text`. I'm aware that I'm fabricating words here and it's wrong grammar, but I wonder if we can just take it as a special nomenclature where "<system cursor name>+able" stands for its might-be-disabled sibling.

Reply:
That makes sense; I hadn't realized what the "able" suffix was really for.